cmd/tkn_tasks.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/func/pkg/pipelines/tekton"
+)
+
+func NewTektonClusterTasksCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tkn-tasks",
+		Short: "List tekton cluster tasks as multi-document yaml",
+		Long: `This command prints tekton tekton task embed in the func binary.
+Some advanced functionality like OpenShift's Web Console build my require installation of these tasks.
+Installation: func tkn-tasks | kubectl apply -f -
```

Comment:
Is manual installation always required? Or the most common use case is covered by creating those resources?

Reply:
AFAIK we only need it when OpenShift's Web Console.

Reply:
Only when you create a function in OCP WebUI you need cluster tasks installed.